scripts/ci-release-app.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# shellcheck source=./common.sh
+source ./scripts/common.sh
+
+set -euo pipefail
+
+EC_VERSION=${EC_VERSION:-}
+APP_VERSION=${APP_VERSION:-}
+APP_CHANNEL=${APP_CHANNEL:-Unstable}
+RELEASE_YAML_DIR=${RELEASE_YAML_DIR:-e2e/kots-release-install}
+REPLICATED_API_ORIGIN=${REPLICATED_API_ORIGIN:-https://api.staging.replicated.com/vendor}
+S3_BUCKET="${S3_BUCKET:-dev-embedded-cluster-bin}"
+
+require S3_BUCKET "${S3_BUCKET:-}"
+require REPLICATED_APP "${REPLICATED_APP:-}"
```

Comment:
how about defaulting to our staging test app and staging origin? seems like the app slug is hardcoded here: https://github.com/replicatedhq/embedded-cluster/pull/1102/files#diff-0bd60a648a661ff431b21e8ea05f4a48bb13f8b2b7f69694d899e007446cf802R45